pages/advanced/cli-reference/common-operations.mdx

```diff
@@ -0,0 +1,253 @@
+
+```bash
+approve_hash <keccak-hexstr-hash> <sender-address>
+```
+
+## Owners
+
+### Add a new owner
+
+Adds a new owner `address` to the Safe.
+
+```bash
+add_owner <address>
+``` 
+
+### Load owners
+
+#### From private key
+
+Loading owners is unnecessary if you want to do `read-only` operations.
+
+To load owners:
+
+```
+> load_cli_owners <account_private_key>
+Loaded account 0xab...cd with balance=123 ether
+Set account 0xab..cd as default sender of txs
+```
+
+You can also load owners from your environment variables before running the Safe CLI:
+
+```bash
+export MY_PRIVATE_KEY=YOUR_EOA_PRIVATE_KEY
+```
+
+Run the Safe-CLI, then:
+
+```
+> load_cli_owners MY_PRIVATE_KEY
+Loaded account 0xab...cd with balance=123 ether
+Set account 0xab..cd as default sender of txs
+```
+
+To check the loaded owners:
+
+```
+> show_cli_owners
+```
+
+To unload an owner:
+
+```
+> unload_cli_owners <ethereum_checksummed_address>
+```
+
+#### From hardware wallets
+
+<Callout type='info' emoji='ℹ️'>
+  Before signing anything, ensure that the data on your hardware wallet device is the same as the Safe CLI data.
+</Callout>
+
+If you want to use both Ledger and Trezor, you need to run the following command:
+
+```bash
+pip install "safe-cli[ledger, trezor]"
+```
+
+##### Ledger
+
+The Ledger module is an optional feature of Safe CLI to sign transactions with the help of [ledgereth](https://github.com/mikeshultz/ledger-eth-lib) library based on [ledgerblue](https://github.com/LedgerHQ/blue-loader-python).
+
+To enable, the Safe CLI must be installed as follows:
+
+```bash
+pip install "safe-cli[ledger]"
+```
+
+When running on Linux, make sure the following rules have been added to `/etc/udev/rules.d/`:
+
+```commandline
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="2c97", ATTRS{idProduct}=="0000", MODE="0660", TAG+="uaccess", TAG+="udev-acl" OWNER="<UNIX username>"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="2c97", ATTRS{idProduct}=="0001", MODE="0660", TAG+="uaccess", TAG+="udev-acl" OWNER="<UNIX username>"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="2c97", ATTRS{idProduct}=="0004", MODE="0660", TAG+="uaccess", TAG+="udev-acl" OWNER="<UNIX username>"
+```
+
+**Ledger commands**
+
+- `load_ledger_cli_owners [--legacy-accounts] [--derivation-path <str>]`: show a list of the first five accounts (--legacy-accounts search using legacy derivation) or load an account from provided derivation path.
+
+##### Trezor
+
+The Trezor module is an optional feature of Safe CLI to sign transactions from the Trezor hardware wallet using the [trezor](https://pypi.org/project/trezor/) library.
+
+To enable, Safe CLI must be installed as follows:
+
+```bash
+pip install "safe-cli[trezor]"
+```
+
+**Trezor commands**
+
+- `load_trezor_cli_owners [--legacy-accounts] [--derivation-path <str>]`: show a list of the first five accounts (--legacy-accounts search using legacy derivation) or load an account from provided derivation path.
+
+
+### Remove an owner
```

Comment:
There are inconsistencies here:
- Remove **an** owner
- Change (the) threhold
- Enable (a) module
- Etc.